packages/invoices/src/InboundShipment/DetailView/api.ts

```diff
@@ -204,3 +218,121 @@ export const getInboundShipmentDetailViewApi = (
     throw new Error(':shrug');
   },
 });
+
+export const useInboundShipment = (): UseQueryResult<Invoice, unknown> => {
```

Comment:
at this point it's just a random assortment of hooks. They're nice and useful, but not a specific interface to follow which might not be such a bad thing, but it makes me sad

Reply:
the hook is a great idea can see this being useful